src/js/structs/__tests__/Deployment-test.js

```diff
@@ -0,0 +1,16 @@
+let Deployment = require('../Deployment');
```

Comment:
if the API changed, we would then change the dummy data we're using and the tests would fail 👍